src/screens/Tests.js

```diff
@@ -1,12 +1,43 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import styles from "./Tests.module.css";
+import TestQuestion from "../components/TestQuestion";
+import TestIntro from "../components/TestIntro";
+
 
 
 function View1() {
+
+  const basicQuestions =[{question: "pytanko 1", answear1: "odp 1", answear2: "odp 2", answear3: "odp 3"},
+                         {question: "pytanko 2", answear1: "odp 1.2", answear2: "odp 2.2", answear3: "odp 3.2"},
+                         {question: "pytanko 3", answear1: "odp 1.3", answear2: "odp 2.3", answear3: "odp 3.3"}
+]
+
+const [questionsToShuffle, setQuestionsToShuffle] = useState(basicQuestions)
+
+  // const displayQuestions = [basicQuestions[Math.floor(Math.random()*basicQuestions.length)]]
+  function shuffle(a) {
+    for (let i = a.length -1; i>0; i--){
+  const j = Math.floor(Math.random()*(i+1));
+  [a[i], a[j]] = [a[j], a[i]];}
+  return a; }
+
+  useEffect(() => {
+  // const toDisplay = shuffle(questionsToShuffle)
+  setQuestionsToShuffle(() => shuffle(questionsToShuffle))
```

Comment:
here it should look different, function shuffle should change the state, something like this:
``` javascript
function shuffle(a) {
    const newQuestions = [...a]
    for (let i = newQuestions.length -1; i>0; i--){
    const j = Math.floor(Math.random()*(i+1));
    [newQuestions[i], newQuestions[j]] = [newQuestions[j], newQuestions[i]];}
    setQuestionsToShuffle(newQuestions)
    return newQuestions; // actually now it's not nessesary now
 }
```

Reply:
``` javascript
useEffect(() => {
   shuffle(basicQuestions)
}, [])

```